ui/app/templates/components/control-group.hbs

```diff
@@ -48,23 +48,25 @@
       </div>
 
       {{#if (or (not this.model.requestEntity) this.currentUserIsRequesting)}}
-        <div class="message is-highlight" data-test-accessor-callout>
```

Comment:
slowly working to remove the `message` css class
<img width="1083" height="455" alt="Screenshot 2025-07-29 at 1 05 57 PM" src="https://github.com/user-attachments/assets/def81820-5741-45e6-92cb-e42c4d7f3ba2" />